candy.bundle.js

```diff
@@ -2027,19 +2027,22 @@ Candy.Core.Event = function(self, Strophe, $) {
 	 * Notify view that the login window should be displayed
 	 *
 	 * Parameters:
-	 *   (String) presetJid - Preset user JID
+	 *   (String) messageKey - i18n Message Key or null
+	 *   (String) presetJid - Preset user JID or null
 	 *
 	 * Triggers:
-	 *   candy:core.login using {presetJid}
+	 *   candy:core.login using {messageKey, presetJid}
 	 */
-    self.Login = function(presetJid) {
+    self.Login = function(messageKey, presetJid) {
         /** Event: candy:core.login
 		 * Triggered when the login window should be displayed
 		 *
 		 * Parameters:
+         *   (String) messageKey - i18n message key
```

Comment:
The indenting shown here is due to tabs being used in the source (not mine i ripped the tab key out of my keyboard years ago:

![keyboard](https://cloud.githubusercontent.com/assets/676479/6547395/03ecb0f6-c5ad-11e4-88ed-a8e6e43aec6c.jpg)